starboard/shared/egl/system_egl.cc

```diff
@@ -57,17 +57,6 @@ SbEglDisplay SbEglGetDisplay(SbEglNativeDisplayType display_id) {
   return eglGetDisplay((EGLNativeDisplayType)display_id);
 }
 
-#if defined(EGL_VERSION_1_5)
```

Comment:
Last review question: the PR as-is completely removes this EGL1.5
function (but note how egl.h includes plenty of EGL data types
e.g. [SbEGLAttrib or SbEGLTime](https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:starboard/egl.h;l=53-63;drc=5e53a763244283242d438df2913de5b7f3a3eeaf) ).

Could we leave this code here and protect the code you added in
fake_graphics_context_provider.cc with an `!defined(EGL_VERSION_1_5)` ?
That way we could  cover both basis -- the majority of our platforms
will have EGL1.5 (even though it might not be technically required)
and will use `eglGetPlatformDisplay()` directly, the rest will try
to use it as an extension.